packages/slice-machine/src/pages/changes.tsx

```diff
@@ -155,6 +171,28 @@ const Changes: React.FunctionComponent = () => {
                 void onPush(false); // not deleting documents by default
               }}
             />
+            <Toast
+              anchor={<Box position="fixed" top={30} right={32} />}
```

Comment:
Position feels weird here, imo, because top and right paddings don't match (personal opinion, btw)

<img width="386" alt="image" src="https://github.com/user-attachments/assets/d823fa50-eba5-4409-bde9-9815205600d8">

Reply:
that's the paddings from Figma, will ask on daily if we can make paddings even :)

Reply:
According to the design, shouldn't the toast be a bit wider?

Reply:
good observation, it was more narrow, but designs changed a little bit during the week, but this change will have to be included in Toast component itself